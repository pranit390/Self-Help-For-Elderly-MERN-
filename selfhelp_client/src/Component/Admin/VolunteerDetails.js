import React from 'react';
import {Button,Container} from 'react-bootstrap' 
import styled from 'styled-components'
import { Link } from "react-router-dom";
import '../../css/home/footer.css'
import AdminNavigationBar from './AdminNav'
import API_URL from "../../ApiHelper/API_URL.js";


const Styles = styled.div`
.container1{
    position:relative;
    width:1200px;
    height:300px;
    margin:240px auto;
}
.boxes{
    position:relative;
    width:calc(400px - 30px);
    height:calc(300px - 30px);
    background: #ccc;
    float:left;
    margin:15px; 
    box-sizing: border-box;
    overflow:hidden;
    border-radius:10px;
}
.h3{
    text-align:center;
    padding:10px;
}
.h5{
    padding:10px;
    text-align:center;
    color:green;
}

 .xyz{
   padding-left: 20px; 
  
   text-align: center; 
   
 }
 

`;

function VolunteerDetails(props) {
    return (
        <Styles>
        <>
        <AdminNavigationBar/>
        <Container>
        <div class="container1">
            <div class="boxes"><h3 class="h3">All Volunteers</h3>
            
            <br/>
             <h5 class="h5">Click here to get the details of all volunteers</h5>
            <div class="xyz">
             <br/>
             <Button variant="light" type="submit" ><Link to="/AllVolunteersDetails" >Click Here</Link></Button>  	   	 
  	 	  	  </div>	  	   
            </div>
            <div class="boxes"><h3 class="h3">Available Volunteers</h3>
            <br/>
             <h5 class="h5">Click here to get all available volunteers</h5>
            <div class="xyz">
             <br/>
             <Button variant="light" type="submit"><Link to="/AvailableVolunteersDetails">Click Here</Link></Button>  	   	 
  	 	  	  </div>
            </div>
            <div class="boxes"><h3 class="h3">Unavailable Volunteers</h3>
            <br/>
             <h5 class="h5">Click here to get all unavailable volunteers</h5>
            <div class="xyz">
             <br/>
             <Button variant="light" type="submit"><Link to="/UnavailableVolunteersDetails">Click Here</Link></Button>  	   	 
  	 	  	  </div>
                
            </div>

        </div>
        
        </Container>
          {/*  <Footer/> */}
        </>
        </Styles>
    );
}

export default VolunteerDetails;