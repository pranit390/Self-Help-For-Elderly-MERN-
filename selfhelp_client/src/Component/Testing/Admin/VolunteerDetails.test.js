import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import {shallow} from 'enzyme'

import VolunteerDetails from '../../Admin/VolunteerDetails.js';

describe("Volunteer Details", () => {
    it("should render my component", () => {
      const wrapper = shallow(<VolunteerDetails/>);
      expect(wrapper.getElements()).toMatchSnapshot();
    });
  });