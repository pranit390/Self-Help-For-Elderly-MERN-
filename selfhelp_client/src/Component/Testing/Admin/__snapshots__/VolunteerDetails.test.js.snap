// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Volunteer Details should render my component 1`] = `
Array [
  <styled.div>
    <React.Fragment>
      <AdminNavigationBar />
      <Container
        fluid={false}
      >
        <div
          class="container1"
        >
          <div
            class="boxes"
          >
            <h3
              class="h3"
            >
              All Volunteers
            </h3>
            <br />
            <h5
              class="h5"
            >
              Click here to get the details of all volunteers
            </h5>
            <div
              class="xyz"
            >
              <br />
              <Button
                active={false}
                disabled={false}
                type="submit"
                variant="light"
              >
                <Link
                  to="/AllVolunteersDetails"
                >
                  Click Here
                </Link>
              </Button>
            </div>
          </div>
          <div
            class="boxes"
          >
            <h3
              class="h3"
            >
              Available Volunteers
            </h3>
            <br />
            <h5
              class="h5"
            >
              Click here to get all available volunteers
            </h5>
            <div
              class="xyz"
            >
              <br />
              <Button
                active={false}
                disabled={false}
                type="submit"
                variant="light"
              >
                <Link
                  to="/AvailableVolunteersDetails"
                >
                  Click Here
                </Link>
              </Button>
            </div>
          </div>
          <div
            class="boxes"
          >
            <h3
              class="h3"
            >
              Unavailable Volunteers
            </h3>
            <br />
            <h5
              class="h5"
            >
              Click here to get all unavailable volunteers
            </h5>
            <div
              class="xyz"
            >
              <br />
              <Button
                active={false}
                disabled={false}
                type="submit"
                variant="light"
              >
                <Link
                  to="/UnavailableVolunteersDetails"
                >
                  Click Here
                </Link>
              </Button>
            </div>
          </div>
        </div>
      </Container>
    </React.Fragment>
  </styled.div>,
]
`;
